authority_test.go

```diff
@@ -34,6 +37,95 @@ var _ = Describe("Authority", func() {
 	Describe("Authority manages issuers", func() {
 		It("creates a new testnet", createNewTestnet)
 
+		It("Authority changes the number of validators", func() {
```

Comment:
Would be nice with a negative test that verifies that a random key cannot change parameters.

Reply:
Done!